src/python/gennylib/genny_runner.py

```diff
@@ -0,0 +1,68 @@
+import sys
+import os
+import subprocess
+from contextlib import contextmanager
+
+def get_genny_args():
+    """
+    Returns the argument list used to create the Genny process.
+
+    If we are in the root of the genny repo, use the local executable. 
+    Otherwise we search the PATH.
+    """
+    args = sys.argv
+    genny_core = 'genny_core'
+
+    local_core = './dist/bin/genny_core'
```

Comment:
Personally I've never been much of a fan of the `dist` directory since if you run `ninja` or `make` again inside `build` after you make changes it doesn't update the stuff inside `dist`. The only way to update `dist` it seems is to run `lamp` again (which unfortunately is hard to make idempotent). Not sure what to suggest other than to start a small fire and slowly homer-simpson-hedge-wall away